### PR TITLE
Suggestions from Pamela

### PR DESCRIPTION
**Pull request by pamelafox**
I'll add inline comments to explain changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,6 @@ To have more understanding of the tables contents. Please login to your Azure SQ
 
 ## Getting Started
 
-### Local Environment
```
Comment (pamelafox) [inline README.md:28]: We should add this heading back when there are Codespaces/DevContainers headings, but for now, it doesnt need a heading.

File: README.md [hunk 2]
```diff
@@ -71,50 +71,46 @@ Once you've opened the project in [Codespaces](#github-codespaces), [Dev Contain
 
 3. Configure your environment variables that will be used for deployment:
 
-> [!IMPORTANT]
-> Please note that SQL Auth cannot be turned off as part of the deployment. However, an admin name and password must be given as part of the database creation process. In the current state, you can enable Microsoft Entra authentication only via the portal.
-
- * aad_admin_name: Principal name of the external administrator (UPN). If you need help finding this value, please login with the Azure CLI and run the script: [fetch-principal-info.sh](./scripts/fetch-principal-info.sh) or [fetch-principal-info.ps1](scripts/fetch-principal-info.ps1)
+    > [!IMPORTANT]
```
Comment (pamelafox) [inline README.md:74]: I happened to reword this section at the same time as you, so I merged our wordings together.

File: README.md [hunk 3]
```diff
@@ -71,50 +71,46 @@ Once you've opened the project in [Codespaces](#github-codespaces), [Dev Contain
...
- * administrator_login: login name of the administrator for the Azure SQL Database.
+    Once you know your principal name, set it as an azd environment variable:
 
     ```shell
-    azd env set AZURE_ADMIN_LOGIN someadminname
+    azd env set AZURE_PRINCIPAL_NAME yourprincipalname
     ```
 
- * administratorPassword: password of the administrator for the Azure SQL Database.
+4. Deploy the resources:
```
Comment (pamelafox) [inline README.md:85]: It's missing azd up right now, I think that got lost in a merge at some point maybe

File: infra/core/ai/openai.bicep [hunk 4]
```diff
@@ -23,7 +22,7 @@ resource openAI 'Microsoft.CognitiveServices/accounts@2023-10-01-preview' = {
     name: openaideploymentname
     sku: {
       name: 'Standard'
-      capacity: 30
+      capacity: chatgpt4oDeploymentCapacity
```
Comment (pamelafox) [inline infra/core/ai/openai.bicep:25]: The capacity variable was used in wrong place, it seemed.

File: infra/main.bicep [hunk 5]
```diff
@@ -10,9 +10,10 @@ param name string
 param tags object = {}
 
 //sql param
-param administratorLogin string
+param administratorLogin string = uniqueString(name, location, subscription().tenantId, 'sqladmin')
```
Comment (pamelafox) [inline infra/main.bicep:13]: I went ahead and generated this, so that people don't have to azd env set it.

**Review by abdulzedan — APPROVED** [review]

**Review by abdulzedan — APPROVED** [review]

**Pull request merged**